rubbery-raccoons/wired/wired_app/views/__init__.py

```diff
@@ -0,0 +1,3 @@
+from . import detail, homepage, search, articleyear, category
+
+__all__ = ["detail", "homepage", "search", "articleyear", "category"]

```

Comment:
`__all__` only applies to `*` imports, so this is unnecessary unless you plan to write `from wired_app.views import *` (which I don't think you should do).

It also doesn't make a ton of sense to "whitelist" submodules like this. Usually `__all__` is used when you have a lot of imports and you only want to expose a few specific items to the end user to be imported.

I recommend deleting this.